MapboxNavigation/CarPlayMapViewController.swift

```diff
@@ -93,6 +90,24 @@ class CarPlayMapViewController: UIViewController, MGLMapViewDelegate {
             mapView.localizeLabels()
         }
     }
+    
+    func resetCamera(animated: Bool = false, defaultAltitude: Bool = false) {
+        let camera = mapView.camera
+        if defaultAltitude {
+            camera.altitude = 16000
+        }
+        camera.pitch = 60
+        mapView.setCamera(camera, animated: false)
+
+    }
+    
+    override func viewSafeAreaInsetsDidChange() {
+        guard isOverviewingRoutes else { return }
```

Comment:
nit: it's good practice to call super on overridden methods unless there’s an explicit reason not to.